kyuubi-util-scala/src/main/scala/org/apache/kyuubi/util/reflect/ReflectUtils.scala

```diff
@@ -37,32 +37,47 @@ object ReflectUtils {
       DynClasses.builder().loader(cl).impl(className).buildChecked()
     }.isSuccess
 
-  def getFieldVal[T](target: Any, fieldName: String): T =
+  def getField[T](target: Any, fieldName: String): T = {
+    val targetClass = target.getClass
     Try {
```

Comment:
the scala `Try` does not help here and may miss FatalError, instead, `try ... catch ...` is more suitable

Reply:
Adopted.